pagopa/io-app
# feat: [PE-552] Revamp CGN merchant details screen

## Short description
This PR implements the new DS on CGN merchant detail screen.

## Design of reference
https://www.figma.com/file/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?type=design&node-id=4405-100549&mode=design&t=YXJlUj57xQtDYAm8-4

**N.B:** The contact section designed inside the Figma with all the merchant contacts is a feature that will be implemented as soon as the backend gives us that information.

## List of changes proposed in this pull request
- Added the new DS components inside the merchant detail screen
- Added a skeleton placeholder while the merchant details are loading

## How to test
- Before you test this pull request, run `yarn install` to update your design system library to the latest one.
- With the CGN card onboarded, navigate through all the flows from the CGN details page until you reach the merchant list, as soon as you tap on it you should be able to see the new design.

## Preview

https://github.com/pagopa/io-app/assets/34343582/6a4602e4-9dc3-4f97-aee2-222db8d73d93
